docs/tools/clients/unity-sdk/samples/nft-example.md

```diff
@@ -384,7 +384,7 @@ import SDKExampleNFT from 0xf8d6e0586b0a20c7
 access(all) fun main(addr:Address): {UInt64:{String:String}} {
 
     //Get a capability to the SDKExampleNFT collection if it exists.  Return an empty dictionary if it does not
-    let collectionCap = getAccount(addr).getCapability<&{SDKExampleNFT.CollectionPublic}>(SDKExampleNFT.CollectionPublicPath)
+    let collectionCap = getAccount(addr).capabilities.get<&{SDKExampleNFT.CollectionPublic}>(SDKExampleNFT.CollectionPublicPath)
     if(collectionCap == nil)
```

Comment:
Since `.get` no longer returns optional, we could probably make this
```suggestion
    if(!collectionCap.check())
```

Or update the line above to use `capabilities.borrow` and remove the `.borrow` below.